build/mark-new-version.sh

```diff
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 NEW_VERSION=${1-}
 
-VERSION_REGEX="v([0-9]+).([0-9]+(.[0-9]+)?)"
+VERSION_REGEX="v([0-9]+).([0-9]+)(.([0-9]+))?"
```

Comment:
Do you mind pulling over https://github.com/GoogleCloudPlatform/kubernetes/blob/master/build/push-official-release.sh#L25 ? I forgot to bonk this one, but I've been meaning to make the patch level prescriptive.